src/main/kotlin/io/snyk/plugin/snykcode/SnykCodeBulkFileListener.kt

```diff
@@ -44,11 +46,13 @@ class SnykCodeBulkFileListener : SnykBulkFileListener() {
         // remove changed files from SnykCode cache
         val allCachedFiles = AnalysisData.instance.getAllCachedFiles(project)
         val filesToRemoveFromCache = allCachedFiles
-            .filter { cachedPsiFile ->
-                val vFile = (cachedPsiFile as PsiFile).virtualFile
-                vFile in virtualFilesAffected ||
+            .filter { cachedFile ->
+                val snykCodeFile = cachedFile as SnykCodeFile
```

Comment:
`toSnykCode`
PS direct cast to PsiFile was a wrong "shortcut" here, shame on me...